docs/stsynphot/appendixa.rst

```diff
@@ -184,9 +184,8 @@ Phoenix Models
 --------------
 
 The ``$PYSYN_CDBS/grid/phoenix`` directory contains models provided by
-`F. Allard et al. <http://perso.ens-lyon.fr/france.allard/>`_
-and can be found in the
-`Star, Brown Dwarf, and Planet Simulator <https://phoenix.ens-lyon.fr/simulator/index.faces>`_. They use static, spherical symmetric, 1D simulations to
+`F. Allard et al. <http://perso.ens-lyon.fr/france.allard/>`_.
+They use static, spherical symmetric, 1D simulations to
```

Comment:
Aha, so this was what was choking the check, thought the error message was very confusing. I'll give this a few days in case they fix the server, otherwise will need to remove the mention of the simulator.

Reply:
I'll wait a bit in case Allard et al. decides to fix the link.